File: resilient/resilient-annotation-processor/src/test/java/ru/tinkoff/kora/resilient/annotation/processor/aop/testdata/CircuitBreakerLifecycle.java
package ru.tinkoff.kora.resilient.annotation.processor.aop.testdata;

import ru.tinkoff.kora.annotation.processor.common.MockLifecycle;

public record CircuitBreakerLifecycle(CircuitBreakerTarget target, CircuitBreakerFallbackTarget fallbackTarget) implements MockLifecycle {
}

File: resilient/resilient-symbol-processor/src/test/kotlin/ru/tinkoff/kora/resilient/symbol/processor/aop/testdata/CircuitBreakerLifecycle.kt
package ru.tinkoff.kora.resilient.symbol.processor.aop.testdata

import ru.tinkoff.kora.annotation.processor.common.MockLifecycle

data class CircuitBreakerLifecycle(val target: CircuitBreakerTarget, val targetFallback: CircuitBreakerFallbackTarget) : MockLifecycle
